### ISSUE TITLE
JetReview - Add Global Configuration

### ISSUE THREAD

**Issue by nightcloud-ar** [Crocoblock/suggestions#581]
Hi.

This suggestion if for JetReviews.

I just bought the plugin after adding at least 400 apartments to my listing web page.

Now I just figure it out that the plugin JetReview it needed to be added BEFORE posting your CPT. 

There should be a way to add this "Globaly" to a PostType even if the PostType is created before. I can't add 400 "Reviews" to each PostType...

Thanks.

**Comment by Lance2j** [comment]
I second this. Actually, making the JetReviews plugin apply to archive posts would be what I'm looking for, so all posts are standardized. I have 2,000 records and another 1,000 on the way. Setting up/changing the review for each post is just not doable.

**Comment by secondsky** [comment]
Indeed, adding every time manually to every post is impossible. I first considered the plugin, but without the option to apply this to a mass of posts its useless for me unfortunately.

**Comment by dankwin** [comment]
This is much needed indeed. I hope this feature will be developed in the near future. It would make JetReviews serious competition for other review plugins.
